Tests/LibWeb/Text/input/DOM/Document-set-alinkColor.html

```diff
@@ -0,0 +1,13 @@
+<body>
```

Comment:
As currently written, your test files don't run correctly. They should look something like this:
```html
<!DOCTYPE html>
<script src="../include.js"></script>
<script>
    test(() => {
        println("before: " + document.bgcolor);
        println("body bgcolor: " + document.body.getAttribute("bgcolor"));    
        
        document.bgcolor = "red";
    
        println("after:" + document.bgcolor);
        println("body bgcolor:" + document.body.getAttribute("bgcolor"));
    });
</script>
```
All of these tests look pretty similar. It might be worth seeing if you can combine them into a single test using a helper function.

---

FYI, you can run the LibWeb tests locally to ensure they run as expected with the following command: 
```sh
./Meta/serenity.sh test lagom LibWeb
```
You should also be able to view the test pages in Ladybird to verify that they look correct.

Reply:
The tests are now combined into one test, but I can't verify if the tests run. When I run the command to run the tests, it says that "no tests were found". I'll try running it on my other computer setup later and see if it works there, but hopefully it is working now after the new changes.

Reply:
That's strange, it *should* just work. Does running `./Meta/serenity.sh run lagom ladybird` first make any difference?

Feel free to ask for help on Discord if you get stuck with this.

Reply:
I got the tests working, I just redownloaded and built everything from scratch and that did the trick.